docs/api-and-integration/compatibility.md

```diff
@@ -6,10 +6,18 @@ position: 130
 
 The table below outlines the backward compatibility between Octopus Server and related components
 
-| Octopus Server   | Octopus.Client Octo.exe | Calamari      | Tentacle     | TeamCity Plugin |
-| --------------   | ----------------------- | ------------  | ------------ | --------------- |
-| 3.2              | 3.2                     | 3.0 ➜ latest | 3.2          |                 |
-| 3.3              | 3.3 ➜ latest           | 3.3           | 3.0 ➜ latest | 3.3 ➜ latest    |
-| 3.4              | 3.3 ➜ latest           | 3.4           | 3.0 ➜ latest | 3.3 ➜ latest    |
-| 3.5 ➜ 2018.2    | 3.3 ➜ latest           | 3.5 ➜ latest | 3.0 ➜ latest | 3.3 ➜ latest    |
-| 2018.2 ➜ latest | 4.30.7 ➜ latest        | 3.5 ➜ latest | 3.0 ➜ latest | 3.3 ➜ latest    |
+| Octopus Server    | Octopus.Client Octo.exe | Calamari         | Tentacle      | TeamCity Plugin  |
+| --------------    | ----------------------- | ------------     | ------------  | ---------------  |
+| 3.2               | 3.2                     | 3.0 ➜ latest    | 3.2           |                  |
+| 3.3               | 3.3 ➜ latest           | 3.3              | 3.0 ➜ latest | 3.3 ➜ latest    |
+| 3.4               | 3.3 ➜ latest           | 3.4              | 3.0 ➜ latest | 3.3 ➜ latest    |
+| 3.5 ➜ 2018.2     | 3.3 ➜ latest           | 3.5 ➜ latest    | 3.0 ➜ latest | 3.3 ➜ latest    |
+| 2018.2 ➜ 2018.12 | 4.30.7 ➜ latest        | 3.5 ➜ latest    | 3.0 ➜ latest | 3.3 ➜ latest    |
+| 2019.1 ➜ latest  | 5.0.0 ➜ latest         | 4.12.1 ➜ latest | 4.0 ➜ latest | coming soon      |
```

Comment:
Maybe put an asterisk next to the Octopus Server version on this row, and an asterisk next to the Note below to tie them together.

Reply:
`Calamari: 4.12.1 ➜ latest`. Shouldn't this be the same as `2018.2 -> 2018.12`? We haven't changed calamari in spaces so I wouldn't expect this version requirement to have changed either.